templates/base.html.twig

```diff
@@ -51,7 +51,7 @@
 </div>
 <div class="navigation-container">
     <div class="navigation">
-        {% for route in ['homepage', 'azure_mfa_registration', 'azure_mfa_authentication', 'gssp_saml_metadata'] %}
+        {% for route in ['homepage', 'gssp_saml_metadata'] %}
```

Comment:
Could we make the pages visible but with some documentation on the page itself?

Reply:
That's the situation we come from I guess. The authentication page was used to also allow us to trigger a bad response. This is also possible from the mock idp. And the registration page still is visible (this is where you fill your email address). 

What would we gain from showing those pages in menu? They are non functional pages when visited outside a SAML authentication flow. They could be filled with text, but that is more valuable for the GSSP example project?

Reply:
Personally for the demo SP I would not mind that they are there as they would let other developers now they exist and are the backbone of the app. I don't see why they need to be hidden in a demo environment.